tests/langchain/test_langchain_autolog.py

```diff
@@ -771,6 +771,19 @@ def test_langchain_autolog_callback_injection_in_batch(invoke_arg, config, async
             assert set(handler.logs) == {"chain_start", "chain_end"}
 
 
+def test_tracing_source_run_in_batch():
+    mlflow.langchain.autolog()
+
+    model = create_openai_llmchain()
+    input = {"product": "MLflow"}
+    with mlflow.start_run() as run:
+        model.batch([input] * 2)
+
+    traces = get_traces()
+    for trace in traces:
+        assert trace.info.request_metadata[TraceMetadataKey.SOURCE_RUN] == run.info.run_id
+
```

Comment:
```suggestion
def test_tracing_source_run_in_batch():
    mlflow.langchain.autolog(log_models=True)

    model = create_openai_llmchain()
    input = {"product": "MLflow"}
    with mlflow.start_run() as run:
        model.batch([input] * 2)

    traces = get_traces()
    for trace in traces:
        assert trace.info.request_metadata[TraceMetadataKey.SOURCE_RUN] == run.info.run_id

    pyfunc_model = mlflow.pyfunc.load_model(run.info.run_id)
    with mlflow.start_run() as run_2:
        pyfunc_model.predict("input")
        
    trace = mlflow.get_last_active_trace()
    assert trace.info.request_metadata[TraceMetadataKey.SOURCE_RUN] == run_2.info.run_id
```

Let's also test the pyfunc prediction.